app/javascript/components/Admin/Purchases/index.tsx

```diff
@@ -8,10 +8,10 @@ import Comments from "$app/components/Admin/Commentable";
 import DateTimeWithRelativeTooltip from "$app/components/Admin/DateTimeWithRelativeTooltip";
 import { Form } from "$app/components/Admin/Form";
 import { NoIcon, BooleanIcon } from "$app/components/Admin/Icons";
+import AdminResendReceiptForm from "$app/components/Admin/Purchases/ResendReceiptForm";
+import { useClientAlert } from "$app/components/ClientAlertProvider";
 import { CopyToClipboard } from "$app/components/CopyToClipboard";
 import { Icon } from "$app/components/Icons";
-import { AdminResendReceiptForm } from "$app/components/server-components/Admin/ResendReceiptForm";
-import { showAlert } from "$app/components/server-components/Alert";
```

Comment:
I don't really understand why we have/need `ClientAlertProvider` at all when `Alert` is a thing - it's in `server-components` because it _can_ be rendered as such, but it doesn't need to be and is used from plenty of client-side React code. Would it make sense to just fix `Alert` if necessary to work in every environment, and then get rid of `ClientAlertProvider` entirely?